tests/platform_tests/api/test_thermal.py

```diff
@@ -54,6 +77,7 @@ def test_get_name(self, duthost, localhost, platform_api_conn):
 
             if self.expect(name is not None, "Unable to retrieve Thermal {} name".format(i)):
                 self.expect(isinstance(name, STRING_TYPE), "Thermal {} name appears incorrect".format(i))
+                compare_value_with_platform_facts(self, 'name', name, i)
```

Comment:
This is a class method. Needs to be prefixed with `self.`

Reply:
fixed